js/pin.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+(function () {
+
+  var mapPins = document.querySelector('.map__pins');
+
+  var mapPinTemplate = document.querySelector('#pin').content.querySelector('.map__pin');
+  var fragment = document.createDocumentFragment();
+
+  var getPinElements = function (template, objectArray) {
+    for (var i = 0; i < window.data.HOTELS_COUNT; i++) {
+      var object = objectArray[i];
```

Comment:
Лучше называть переменные более конкретно: 
```suggestion
      var pin = objectArray[i];
```